command/secrets/generate/secrets_generate.go

```diff
@@ -51,29 +49,26 @@ func setFlags(cmd *cobra.Command) {
 		typeFlag,
 		string(secrets.HashicorpVault),
 		fmt.Sprintf(
-			"the type of the secrets manager. Default: %s",
+			"the type of the secrets manager. Available types: %s, %s, %s and %s",
+			secrets.Local,
```

Comment:
Please note that we don't use `secrets generate` for the local file system key management.
We use `secrets init` for that purpose, this generate command is only for generating the config

Reply:
Got it. Should we remove the support for `--type local` from the code completely ? It seams redundant...

Reply:
Removed 981bd6d59617f854b8e2e991ee2f88a20e714e41